### PR TITLE
DEV2-2244 ci onprem

### PR DESCRIPTION
**Pull request by avnerbarr**

### CODE REVIEW

File: .github/workflows/onPremPackage.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: On Prem Package
+on:
+  workflow_dispatch:
+    inputs:
+      binaryVersion:
+        type: string
+        description: the version of the tabnine binary which will be packaged into this plugin
+        required: true
+      plugInVersion:
```
Comment (assapir) [inline .github/workflows/onPremPackage.yml:9]: just use the version out there...

File: .github/workflows/onPremPackage.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        with:
+          java-version: 11
+      - uses: "google-github-actions/auth@v1"
+        with:
+          credentials_json: ${{ secrets.GCS_RELEASE_KEY }}
+      - uses: 'google-github-actions/setup-gcloud@v1'
+      - name: Build
+        id: build
+        run: |
+         echo "downloading binaries"
```
Comment (assapir) [inline .github/workflows/onPremPackage.yml:30]: do the `sed` thingy?

**Review by assapir — APPROVED** [review]

File: .github/workflows/onPremPackage.yml (hunk 2 above)
Comment (avnerbarr) [reply .github/workflows/onPremPackage.yml:30]: I need to test it. no point changing anything yet

**Review by yanir-codota — APPROVED** [review]
approved

**Pull request merged**